plenum/common/ledger_manager.py

```diff
@@ -89,11 +90,18 @@ def addLedger(self, ledger_id: int, ledger: Ledger,
             preCatchupStartClbk=preCatchupStartClbk,
             postCatchupCompleteClbk=postCatchupCompleteClbk,
             postTxnAddedToLedgerClbk=postTxnAddedToLedgerClbk,
-            verifier=MerkleVerifier(ledger.hasher)
+            verifier=MerkleVerifier(ledger.hasher),
+            taa_acceptance_required=taa_acceptance_required
         )
 
         self._node_leecher.register_ledger(ledger_id)
 
+    def ledger_info(self, lid: int) -> Optional[LedgerInfo]:
+        try:
+            return self.ledgerRegistry[lid]
```

Comment:
Why don't we use `self.ledgerRegistry,get(lid)` instead?

Reply:
thanks, updated